io/zenoh-transport/tests/unicast_authenticator.rs

```diff
@@ -606,6 +606,18 @@ fn authenticator_udp() {
     task::block_on(run(&endpoint));
 }
 
+#[cfg(feature = "transport_shm")]
+#[test]
+fn authenticator_shm() {
+    let _ = env_logger::try_init();
+    task::block_on(async {
+        zasync_executor_init!();
+    });
+
+    let endpoint: EndPoint = "shm//tmp/authenticator_shm_test".parse().unwrap();
+    task::block_on(run(&endpoint));
+}
+
 #[cfg(feature = "transport_ws")]
 #[test]
```

Comment:
Same applies here (which slipped from some old code):.

I believe there is no need to run this test every time on the CI (it will make increase its duration).
Mark the test as follows:
```
#[test]
#[ignore]
```
It can be then run with `cargo test -- --ignored`.